### PR TITLE
Protocol timeline

### PR DESCRIPTION
**Pull request by jasonLaster**
https://www.loom.com/share/0a44bda4931c4828839fcfab00d950ec

Screen Shot 2022-05-14 at 7 54 14 AM
Screen Shot 2022-05-14 at 7 54 19 AM

### CODE REVIEW

File: src/ui/components/Timeline/index.tsx [hunk 1]
```diff
@@ -101,7 +103,7 @@ export default function Timeline() {
   return (
     <>
       <FocusModePopout />
-      <div className="timeline">
+      <div className={`timeline ${showProtocolTimeline ? "show-protocol-timeline" : ""}`}>
```
Comment (jcmorrow) [inline src/ui/components/Timeline/index.tsx:106]: I prefer how `classNames` would read here, but don't feel strongly about it.

File: src/ui/components/Timeline/Tooltip.css [hunk 2]
```diff
@@ -7,8 +7,11 @@
   align-items: center;
   justify-content: center;
   z-index: var(--z-index-1--timeline-tooltip);
-  background: var(--chrome);
-  padding: 1px;
+  background: var(--body-bgcolor);
+  padding: 3px 5px;
+  border-radius: 4px;
+
```
Comment (jcmorrow) [inline src/ui/components/Timeline/Tooltip.css:13]: Is this newline meaningful?

File: src/ui/components/Timeline/Timeline.css [hunk 3]
```diff
@@ -33,6 +33,10 @@
   font-size: 29px;
 }
 
+.timeline.show-protocol-timeline {
+  padding-top:40px;
```
Comment (jcmorrow) [inline src/ui/components/Timeline/Timeline.css:37]: This feels a bit janky from a layout perspective, but I also don't want to be a perfectionist, especially for what is basically an internal diagnostic tool 🔨

**Review by jcmorrow — APPROVED** [review]

File: src/ui/components/Timeline/Timeline.css (hunk 3 above)
Comment (jasonLaster) [reply src/ui/components/Timeline/Timeline.css:37]: hehe totally. This is a surprisingly hard problem

File: src/ui/components/Timeline/index.tsx (hunk 1 above)
Comment (jasonLaster) [reply src/ui/components/Timeline/index.tsx:106]: me too - but lazy

File: src/ui/components/Timeline/Timeline.css (hunk 3 above)
Comment (bvaughn) [reply src/ui/components/Timeline/Timeline.css:37]: :cough:Tailwind:cough: 😇

**Pull request merged**